addons/website_event_questions/models/event_question.py

```diff
@@ -26,7 +26,7 @@ class EventQuestion(models.Model):
     @api.constrains('event_type_id', 'event_id')
     def _constrains_event(self):
         if any(question.event_type_id and question.event_id for question in self):
-            raise UserError(_('Question cannot belong to both the event category and itself.'))
+            raise ValidationError(_("Question cannot be linked to both an Event and an Event Type."))
```

Comment:
OOC why changing the error type ? I know constraints should raise ValidationError but you could break external code relying on an UserError for example. Does that solve something ? Otherwise we should keep the diff as minimal as possible.

Reply:
If it's preferable to keep the wrong type of error over changing it, then yeah, we can leave it as is.